API.md

```diff
@@ -1,64 +1,272 @@
+    - [finally()](#finally)
+    - [afterJSON()](#afterjson)
+    - [only()](#only)
+    - [timeout(ms)](#timeoutms)
+    - [retry(count, backoff)](#retrycount-backoff)
+    - [baseUri(uri)](#baseuriuri)
+    - [waits(ms)](#waitsms)
+    - [exceptionHandler(function)](#exceptionhandlerfunction)
+  - [Inspectors](#inspectors)
+    - [inspect(cb)](#inspectcb)
+    - [inspectRequest(message)](#inspectrequestmessage)
+    - [inspectResponse(message)](#inspectresponsemessage)
+    - [inspectHeaders(message)](#inspectheadersmessage)
+    - [inspectJSON(message)](#inspectjsonmessage)
+    - [inspectBody(message)](#inspectbodymessage)
+    - [inspectStatus(message)](#inspectstatusmessage)
+    - [Send Raw JSON or POST Body](#send-raw-json-or-post-body)
+
+## The Basics
+
+Every frisby request begins with `create(..)` and ends with a `toss()`.
+
+### create(msg)
+
+- Types: `msg`: `string`
+- Default: `none`
+
+Used to create an instance of IcedFrisby that is used to send 1 request and receive the response.
+The `msg` is used to name the test when it's wrapped in mocha for execution at runtime.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+### toss()
+
+Used to complete the list of commands and issue the request.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+## Commands
+
+These are the commands you'll need to get IcedFrisby talking HTTP.
+
+An optional parameters object is accepted in all of these methods, but aren't _useful_ for every verb. All object parameters are optional.
+
+```javascript
+{
+  json: 'boolean',         //Whether this will be a JSON body. Overrides value set in globalSetup().
+  body: 'string'|'object', //The body to include in the outbound request. This overrides "data" if provided in method call.
+  mock: 'function',        //A mock runner to use. When not provided, uses "request" (i.e. does it for real).
+  form: 'boolean'          //Use the object in the body to create a form-encoded request.
+}
+```
+
+### get(uri, params)
+
+- Types: `uri`: `string`, `params`: `object` (optional)
+- Default: `none`
+
+Perform an HTTP GET on the specified URI.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com/login')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+### head(uri, params)
+
+Identical to [get](#geturi-params), using an HTTP HEAD request.
+
+### options(uri, params)
+
+Identical to [get](#geturi-params), using an HTTP OPTIONS request.
+
+### post(uri, data, params)
+
+- Types: `uri`: `string`, `data`:`object` `params`: `object` (optional)
+- Default: `none`
+
+Perform an HTTP POST to the specified URI.
+
+```javascript
+frisby.create('a test')
+    .post('http://example.com/login', {
```

Comment:
It might be slightly clearer to provide a more REST-like example. How about making this post to `https://example.com/account`?

Reply:
I have no strong feelings on this. Updated.